baseframe/forms/validators.py

```diff
@@ -309,9 +315,10 @@ class IsEmoji(object):
     :param message:
         Error message to raise in case of a validation error.
     """
+    default_message = __(u"This is not a valid emoji.")
```

Comment:
Remove trailing period in this message. The others don't have it. Also audit for other trailing periods. The rule is this:

1. If the message is a single statement, no trailing period.
2. If the message has two (or more) statements, use a period at the end of each statement except the last one.
3. If the message ends in other punctuation such as ! or ?, leave those in. _Only_ the last full stop is to be removed.